practiceCourseGenerator/practiceCourseGenerator.py

```diff
@@ -0,0 +1,233 @@
+'''
+
+@author: Kyle Dove
+
+Purpose: Download users list from Google Drive. Generate CSV files in 
+directory. Zip directory to be used to create Canvas practice courses.
+
+DATE           Author               Description   
+===========    ===============      ============================================
+Mar 31 2015    Kyle Dove			TLUNIZIN-470
+									Created.
+
+'''
+
+import logging
+import time
+import csv
+import requests
+import json
+import os
+import zipfile
+from apiclient.discovery import build
+from httplib2 import Http
+from oauth2client import file, client, tools
+from apiclient import errors
+from apiclient import http
+
+def download_file(service, drive_file):
```

Comment:
This is a minor preference on my part, but I'd recommend using camelCase for all the symbols you define.  I'd rather not have all the extra underscores around.  All caps for constant-like variables is good.  There's probably a Python style guide somewhere that may be helpful.